styles/stylesheet.css

```diff
@@ -0,0 +1,296 @@
+* {
+  box-sizing: border-box;
+}
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+body {
+  min-height: 100%;
+  display: grid;
+  grid-template-rows: 1fr auto;
+  font-family: 'Arial', sans-serif;
+}
+body a {
+  text-decoration: none; /* removes hyperlink underlines */
+}
+body a:hover {
+  color: black;
+  text-decoration: underline;
+}
+/********** MEDIA QUERIES **********/
+/* for phones - dog grid layout needs to change from 3 columns to 1 (home) and hide blog aside */
+@media not print and (min-width: 576px) and (max-width: 767px) {
+  .main-content {
+    display: flex;
+    /* flex-flow: row wrap;
+    margin: 2rem 4rem; */
+    width: 80%;
+    margin: 2rem auto;
+  }
+  .main-content-blog {
+    display: flex;
+    flex-flow: row wrap;
+    margin: 2rem 6rem;
+  }
+  .home-content {
+    display: grid;
+    flex: 70%;
+    align-items: center;
+    margin-bottom: 2rem;
+  }
+  .blog-aside {
+    /* having trouble figuring out why this part isn't working */
+    display: none;
+    flex: 30%;
+    flex-flow: row wrap;
+    margin: -1rem 6rem 2rem 4rem;
+  }
+  .page-content {
+    grid-gap: 2rem;
+    display: flex;
+    flex-flow: row wrap;
+    margin-left: 2rem;
+  }
+  .page-content-dogs {
+    grid-area: 2 / 1 / 3 / 4;
```

Comment:
Your dog tiles on your home page don't appear to be scaling. I suggest looking at using css grid, specifically auto-fit and minmax for the tile sizes. See MDN doc here: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout